chia/rpc/wallet_rpc_api.py

```diff
@@ -1500,9 +1507,18 @@ async def nft_get_nfts(self, request) -> EndpointResult:
         wallet_id = uint32(request["wallet_id"])
         nft_wallet: NFTWallet = self.service.wallet_state_manager.wallets[wallet_id]
         nfts = nft_wallet.get_current_nfts()
+        start_index = request.get("start_index", 0)
+        num = request.get("num", len(nfts))
```

Comment:
I appreciate building out support for pagination. One GUI use case that this RPC doesn't natively handle is fetching _all_ NFTs across all wallets. At the moment, the GUI is calling this RPC for each NFT wallet and aggregating the results into a single list.

I can see this pagination implementation being useful when selecting a specific wallet in the GUI/CLI, but if "All NFTs" is selected, this form of pagination is not going to work.

Would it be too much trouble to add a mode here to fetch all NFTs? Perhaps we could just make the `"wallet_id"` param optional -- if None, fetch all NFTs.

Reply:
I think it should be as simple as adding an outer loop that iterates over NFT wallets